core/arch/arm/plat-marvell/platform_config.h

```diff
@@ -42,7 +42,7 @@
 #endif /*ARM64*/
 
 /* SDP enable but no pool defined: reserve 4MB for SDP tests */
-#if defined(CFG_SECURE_DATA_PATH) && !defined(CFG_TEE_SDP_MEM_BASE)
+#ifndef CFG_TEE_SDP_MEM_BASE
```

Comment:
I think this will not match. The issue is that `CFG_TEE_SDP_MEM_TEST_SIZE` impacts the memory layout, hence its value set to 0 when SDP is disabled or `CFG_TEE_SDP_MEM_BASE` already enabled.
This comment applies to some platforms marvell, vexpress (qemu_virt/qelu_armv8) and stm.

Reply:
@etienne-lms So use original code and don't make any changes for these platforms?

Reply:
Yes, i think it's better, as @jforissier suggested.